docs/monitoring/metrics.md

```diff
@@ -671,6 +673,8 @@ metrics.reporter.influxdb.db: flink
 metrics.reporter.influxdb.username: flink-metrics
 metrics.reporter.influxdb.password: qwerty
 metrics.reporter.influxdb.retentionPolicy: one_hour
+metrics.reporter.influxdb.connectTimeout: 10000
+metrics.reporter.influxdb.writeTimeout: 10000
```

Comment:
This is an example configuration, pasting the values that match default values is redundant.
I propose to either remove them, or use some different values as samples.